### PR TITLE
Feature nodejs wrapper update

### PR DESCRIPTION
**Pull request by skynetsib**
New route added getAbTestCampaignResult
New route added getSharedTemplateUrl
New route added sendEmailCampaignNow
New route added getChildAccountCreationStatus
New route added getTransacBlockedContacts
New route added getTransacEmailContent
New route added getTransacEmailsList
New route added smtpBlockedContactsEmailDelete
New route added smtpLogMessageIdDelete

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,16 +9,18 @@ SendinBlue's API matches the [OpenAPI v2 definition](https://www.openapis.org/).
...
+For more information, please visit [https://account.sendinblue.com/support](https://account.sendinblue.com/support)
 
 ## Installation
 
 ### For [Node.js](https://nodejs.org/)
 
 #### npm
 
-The following recommended installation requires [npm](https://npmjs.org/). If you are unfamiliar with npm, see the [npm docs](https://npmjs.org/doc/).
+To publish the library as a [npm](https://www.npmjs.com/),
```
Comment (aakanksha-sib) [inline README.md]: These 2 lines need to stay as they were before
`The following recommended installation requires [npm](https://npmjs.org/). If you are unfamiliar with npm, see the [npm docs](https://npmjs.org/doc/).`

File: README.md [hunk 2]
```diff
@@ -60,6 +62,19 @@ then install it via:
     npm install sendinblue/APIv3-nodejs-library --save
 ```
 
+### For browser
```
Comment (aakanksha-sib) [inline README.md]: Until unless you have tested these steps, don't publish it here.

File: README.md [hunk 3]
```diff
@@ -388,27 +425,15 @@ Class | Method | HTTP request | Description
 
 ## Documentation for Authorization
```
Comment (aakanksha-sib) [inline README.md]: The `Documentation for Authorization` part should remain as before. Also, please add back the `Support and Feedback` section

File: package.json [hunk 4]
```diff
@@ -1,33 +1,17 @@
...
-  "bugs": {
-    "url": "https://github.com/sendinblue/APIv3-nodejs-library.git/issues"
-  },
-  "homepage": "https://github.com/sendinblue/APIv3-nodejs-library.git/",
   "browser": {
     "fs": false
   },
   "dependencies": {
-    "superagent": "3.7.0"
+    "superagent": "3.5.2"
```
Comment (aakanksha-sib) [inline package.json]: Is it deliberately degraded ?

Comment (skynetsib) [reply package.json]: Its automatically degraded on generating wrapper

**Pull request merged**